### DISCUSSION TITLE
How to extract the section lift coefficient of the wing in wing.jl

### DISCUSSION THREAD

**Discussion by inse0918** [byuflowlab/FLOWUnsteady#61] [Q&A] [answered]
Hi.

Thank you for your efforts first.

When the wing.jl code is done, the results of the section lift coefficient of an wing saved at the 'wing-example_convergence.png', like this
![wing-example_convergence](https://user-images.githubusercontent.com/93040154/233913275-2808201a-817c-439e-b4e7-eaf4bfaab41f.png)

So, I want to compare the section lift coefficient that I ran with reference data.

But, I cannot find the section aerodynamic coefficient data when the code running.

So I want to save the section aerodynamic coefficient of an wing when the last time step or everything.

Where is the section aerodynamic data in the wing.jl code ?
or How can I handle the section aerodynamic data for saving the cl data on where I want ?

Thank you.
Best regards,
Inseo.

**Comment by EdoAlvarezR** [comment]
Hi Inseo, 

Please see the full example under [`examples/wing/`](https://github.com/byuflowlab/FLOWUnsteady/tree/master/examples/wing) to see how to postprocess the simulation and obtain the plots shown in the tutorial:

![image](https://user-images.githubusercontent.com/14857108/234367704-22adfdc0-0c3c-4030-b3ad-3a2921777c5f.png)
![image](https://user-images.githubusercontent.com/14857108/234367727-60898fb3-ed83-420c-b970-8452768fd0a1.png)

**Comment by inse0918** [comment]
Hi Ed.

Thank you for your reply.

The I had already check the validation case that you told me, Thank you.

In specific, I want to extract the data type of the sectional lift coefficient(cl) like this.

for example,

**Span position 2y/b 0.0 0.2 0.4 0.6 0.8 1.0
Sectional lift cl 0.34 0.35 0.36 0.35 0.33 0.21**

Like this data type(*.dat, *.txt, else.)(not a plot sorry...).

Is there way to extract raw data of the sectional cl or cd in code ?

Thank you.
Best regards,
Inseo.

**Comment by EdoAlvarezR** [comment]
Yes, you can save the data that you are plotting in any format you would like. You can simply [print it to a text file](https://en.wikibooks.org/wiki/Introducing_Julia/Working_with_text_files), or [save it as a CSV](https://github.com/JuliaData/CSV.jl).

**Reply by inse0918** [reply]
Hi Ed.

Thank you for your kind reply always ! !

Then, could you tell me the variable where the section aerodynamic coefficient(like Cl, Cd and Gamma) is saved in source code ?

Thank you.
Best regards,
Inseo

**Reply by EdoAlvarezR** [reply, marked as answer]
Here is the exact line where things get plotted: [LINK](https://github.com/byuflowlab/FLOWUnsteady/blob/master/examples/wing/wing_aoasweep.jl#L149). Instead of plotting, you can save `cl` and `cd` to a file.

**Reply by inse0918** [reply]
Hi Ed.

Thank you for your clear answer !

Finally, I can handle the section aerodynamic data using cl and cd.

Here is the validation results of the isolated wing of the PROWIM[Veldhuis]

![image](https://user-images.githubusercontent.com/93040154/235091348-2d5d84ac-cb5b-4838-a6ba-d6ab2948212e.png)

Thank you.
Best regards,
Inseo
